scripts/buildsystems/msbuild/vcpkg.targets

```
@@ -70,6 +68,7 @@
 
     <_ZVcpkgConfigSubdir Condition="'$(_ZVcpkgNormalizedConfiguration)' == 'Debug'">debug\</_ZVcpkgConfigSubdir>
     <_ZVcpkgExecutable>$(_ZVcpkgRoot)vcpkg.exe</_ZVcpkgExecutable>
+    <ExternalIncludePath>%(ExternalIncludePath);$(_ZVcpkgCurrentInstalledDir)include</ExternalIncludePath>
```

```suggestion
    <ExternalIncludePath>$(ExternalIncludePath);$(_ZVcpkgCurrentInstalledDir)include</ExternalIncludePath>
```

This is a property, not an item expansion, so I think you're clobbering existing settings of this property with a bogus string before.

---

Please double check; if you like the suggestion let me know and we'll merge this with that.

---

Ah yes, you're right. Thanks for the fix!